docs/sections/user_guide/cli/drivers/chgres_cube.rst

```diff
@@ -6,13 +6,15 @@ The ``uw`` mode for configuring and running the :ufs-utils:`chgres_cube<chgres-c
 .. code-block:: text
 
    $ uw chgres_cube --help
-   usage: uw chgres_cube [-h] TASK ...
+   usage: uw chgres_cube [-h] [--version] TASK ...
 
    Execute chgres_cube tasks
 
    Optional arguments:
      -h, --help
          Show help and exit
+     --version
+         Show version info exit
```

Comment:
Should this say "show version info and exit"?

Reply:
Thank you, will fix.

Reply:
Oh, of course -- I fixed the _source_ of the problem, but need to fix all the captured CLI output in the docs! I can't wait for that automation...